tools/model-compiler/src/test/java/io/spine/tools/compiler/check/DependencyConfigurerTest.java

```diff
@@ -21,91 +21,101 @@
 package io.spine.tools.compiler.check;
 
 import com.google.common.testing.NullPointerTester;
+import io.spine.testing.SlowTest;
 import org.gradle.api.Project;
 import org.gradle.api.artifacts.Configuration;
 import org.gradle.api.artifacts.ConfigurationContainer;
 import org.gradle.api.artifacts.Dependency;
 import org.gradle.api.artifacts.DependencySet;
-import org.junit.jupiter.api.BeforeEach;
+import org.gradle.api.internal.artifacts.dependencies.DefaultExternalModuleDependency;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
-import java.util.Optional;
-
+import static com.google.common.truth.Truth.assertThat;
+import static io.spine.tools.compiler.check.DependencyConfigurer.MODEL_COMPILER_PLUGIN_NAME;
 import static io.spine.tools.compiler.check.DependencyConfigurer.SPINE_CHECKER_MODULE;
 import static io.spine.tools.gradle.Artifact.SPINE_TOOLS_GROUP;
 import static io.spine.tools.gradle.ConfigurationName.annotationProcessor;
-import static io.spine.tools.gradle.compiler.given.ModelCompilerTestEnv.newProject;
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.Mockito.spy;
-import static org.mockito.Mockito.when;
-
-@SuppressWarnings({"CheckReturnValue", "ResultOfMethodCallIgnored"})
+import static io.spine.tools.gradle.ConfigurationName.classpath;
+import static io.spine.tools.gradle.compiler.given.Project.newProject;
+
+// TODO:2019-09-15:dmytro.kuzmin: This test configures the project with real dependencies and
+//  repositories which leads to a slow test execution. In future, it should be removed in favor of
+//  proper integration tests for the `spine-errorprone-checks` plugin.
+@SuppressWarnings("CheckReturnValue")
 // We ignore boolean "success" flag which is not interesting for us in this test.
+@SlowTest
 @DisplayName("DependencyConfigurer should")
 class DependencyConfigurerTest {
 
-    private static final String STUB_VERSION = "versionStub";
-
-    private Configuration annotationProcessorConfig;
-    private DependencyConfigurer helper;
-    private DependencyConfigurer helperMock;
+    private static final String SPINE_VERSION = "1.1.0";
```

Comment:
Can we read this property from the `version.gradle` file? It will never be obvious this file references the version that has to be updated each release.

Besides, the `releases` repository is also specific to this version. The `SNAPSHOT` versions won't resolve properly.